docs/guide/content_model.md

```diff
@@ -169,9 +169,12 @@ Aside from the Field Type, the Field definition in a Content Type provides the f
 
 **Description** – a detailed description of the Field.
 
-**Required** – a flag which indicates if the Field is required for the system to accept the Content item. In other words, if a Field is flagged as Required, a user will not be able to save or publish a Content item without filling in this Field.
+**Required** – a flag which indicates if the Field is required for the system to accept the Content item. By default, if a Field is flagged as Required, a user will not be able to publish a Content item without filling in this Field.
+
+!!! note
+    You can use the `ContentService::validate()` method to decide whether the required fields or whole Content items are checked for completeness at other stages of the workflow.
```

Comment:
```suggestion
    You can use the `ContentService::validate()` method to decide whether the required Fields or whole Content items are checked for completeness at other stages of the workflow.
```
Does this refer to "workflow" as in "Editorial workflow"? If not, it would be be better to rephrase it, it can be misleading.

Reply:
It is about a regular workflow that a Content item can go through, not necessarily the one available in Enterprise.